renormalizer/mps/mps.py

```diff
@@ -1566,3 +1622,15 @@ def __str__(self):
     # todo: not used?
     def __iter__(self):
         return iter((self.bra_mps, self.ket_mps))
+
+
+def min_abs(t1, t2):
+    # t1, t2 could be int, float, complex
+    # return the number with smaller norm
+
+    if xp.iscomplex(t1) != xp.iscomplex(t2):
+        raise TypeError(f"{xp.iscomplex(t1)} != {xp.iscomplex(t2)}")
```

Comment:
I suspect the error message would be some like `True != False` which is not very helpful.

Reply:
I've fixed it. Please see the new update.